tensorrt_llm/llmapi/llm_args.py

```diff
@@ -850,12 +850,18 @@ class CacheTransceiverConfig(BaseModel, PybindMirror):
     """
     Configuration for the cache transceiver.
     """
-    max_num_tokens: Optional[int] = Field(
+    backend: Optional[str] = Field(
```

Comment:
If the `backend` can only accept a set of strings, it is better to typing it as `Literal`, e.g. `backend: Literal["default", "pytorch", "tensorrt"]`, this should help guide the user about how to set it.